### DISCUSSION TITLE
SQLite

### DISCUSSION THREAD

**Discussion by ktk** [ontop/ontop#505] [Q&A] [answered]
Hi everyone,

Quick question, has anyone used Ontop with SQLite? I have some data in SQLite on my system that I would like to have in RDF.

It does not seem to be listed as one of the supported DBs in the documentation.

regards

adrian

**Comment by bcogrel** [comment, marked as answer]
Hi Adrian, 

Indeed, SQLite is not yet supported, as there are no dedicated factories for it at the moment (https://ontop-vkg.org/dev/db-adapter). This feature request is tracked by https://github.com/ontop/ontop/issues/134.

Best,
Benjamin
